### DISCUSSION TITLE
Zeek intel (OTX & Misp) not working after 2.3.40 upgrade

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#3668] [Unsupported Versions] [answered]
Using the 2.3.30 ISO, OTX & misp intel hits were properly shown on Hunt intel & Kibana. Not changing any config file, I get no hit anymore since the upgrade to 2.3.40 despite testing nslookups to Intel::Domain entries. 
intel.log is inexistent since the upgrade, reporter.log doesn't show any error and loaded_scripts shows otx and intel zeek loaded. 
A fresh 2.3.40 install in a VM shows the same problem while a fresh 2.3.30 install shows intel hits working.

**Comment by weslambert** [comment]
I'll need to test this and see if I can duplicate, then get back with you.

**Comment by bdorr1105** [comment]
I am actually having the same problem, and it is not with MISP, but zeek intel in general, so maybe this whole situation is related. I checked the reporter.log and it is blank. It is not generating a single intel.log and I put a generic 8.8.8.8 and conducted a ping, still no hits

**Comment by dougburks** [comment, marked as answer]
We've confirmed a bug in Zeek 4.0.0 that prevents Zeek Intel from working properly:
https://github.com/Security-Onion-Solutions/securityonion/issues/3850
